bin/node/cli/src/service.rs

```diff
@@ -719,6 +721,7 @@ mod tests {
 				let check_nonce = frame_system::CheckNonce::from(index);
 				let check_weight = frame_system::CheckWeight::new();
 				let payment = pallet_transaction_payment::ChargeTransactionPayment::from(0);
```

Comment:
This needs to be removed. `ChargeAssetTxPayment` replaces `ChargeTransactionPayment`